api/src/main/java/uk/gov/hmcts/payment/api/configuration/RestTemplateConfiguration.java

```diff
@@ -29,16 +40,38 @@ public RestTemplate restTemplatePaymentGroup() {
         return new RestTemplate(new HttpComponentsClientHttpRequestFactory());
     }
 
+    @Bean (value = "restTemplateLiberata")
+    public RestTemplate restTemplateLiberata() {
+        int readTimeout = Integer.parseInt(liberataReadTimeout);
+        int connectTimeout = Integer.parseInt(liberataConnectTimeout);
+
+        PoolingHttpClientConnectionManager connectionManager = setupConnectionManager(readTimeout, connectTimeout);
+
+        CloseableHttpClient httpClient = HttpClients.custom()
+            .setConnectionManager(connectionManager)
+            .build();
+
+        HttpComponentsClientHttpRequestFactory factory = new HttpComponentsClientHttpRequestFactory(httpClient);
+        return new RestTemplate(factory);
+    }
+
     @Bean (value = "restTemplateRefundsGroup")
     public RestTemplate restTemplateRefundsGroup() {
         return new RestTemplate(new HttpComponentsClientHttpRequestFactory());
     }
 
     @Bean (value = "restTemplateIacSupplementaryInfo")
     public RestTemplate restTemplateIacSupplementaryInfo() {
```

Comment:
I imagine, you could create a method to have common code for this and `restTemplateLiberata()` methods, e.g.:
```
@Bean(value = "restTemplateIacSupplementaryInfo")
public RestTemplate restTemplateIacSupplementaryInfo() {
   int readTimeout = Integer.parseInt(iacReadTimeout);
   int connectTimeout = Integer.parseInt(iacConnectTimeout);
   return createRestTemplate(readTimeout, connectTimeout);
}

private RestTemplate createRestTemplate(int readTimeout, int connectTimeout) {
    PoolingHttpClientConnectionManager connectionManager = setupConnectionManager(readTimeout, connectTimeout);
    CloseableHttpClient httpClient = HttpClients.custom()
            .setConnectionManager(connectionManager)
            .build();

    HttpComponentsClientHttpRequestFactory factory = new HttpComponentsClientHttpRequestFactory(httpClient);
    return new RestTemplate(factory);
}
```